tests/sentry/api/endpoints/test_sentry_internal_app_tokens.py

```diff
@@ -106,5 +106,45 @@ def test_token_is_masked(self):
         response = self.client.get(url, format="json")
         response_content = json.loads(response.content)
 
+        assert len(response_content) == 1
         assert response_content[0]["token"] == MASKED_VALUE
         assert response_content[0]["refreshToken"] == MASKED_VALUE
+
+    def test_no_scope_api_token(self):
+        user = self.create_user(email="meep@example.com")
+        self.create_member(organization=self.org, user=user, role="owner")
+        sentry_app = self.create_internal_integration(
+            name="sentry_app_1",
+            organization=self.org,
+            scopes=(
+                "project:read",
+                "project:write",
+                "project:admin",
+                "project:releases",
+                "team:read",
+                "team:write",
+                "team:admin",
+                "event:read",
+                "event:write",
+                "event:admin",
+                "org:read",
+                "org:write",
+                "org:admin",
+                "member:read",
+                "member:write",
+                "member:admin",
+            ),
+        )
+        sentry_app_api_token_2 = ApiToken.objects.create(
```

Comment:
Can we give this a more descriptive name? Also I don't think there is a `sentry_app_api_token_1`